.github/workflows/test.yml

```diff
@@ -110,13 +112,15 @@ jobs:
       run: |
         Add-Content -Path C:\Windows\System32\drivers\etc\hosts -Value "1.2.3.4 xn--fa-hia.de"
         Add-Content -Path C:\Windows\System32\drivers\etc\hosts -Value "5.6.7.8 fass.de"
+    - name: Install uv
+      run: pip install uv
```

Comment:
Why not using https://github.com/astral-sh/setup-uv?

Reply:
I forgot it existed. I'll change the PR to use that.